src/guest/start.c

```diff
@@ -379,6 +379,37 @@ static int set_available_vf(void)
 	char buf[64] = { 0 };
 	int i;
 	int ret = 0;
+	int wst;
```

Comment:
This is not required for all the cases. vfio is only required to passthrough devices.

Reply:
and even it is really needed, please use load_kernel_module("vfio") and load_kernel_module("vfio-pci")